docs/root/intro/version_history.rst

```diff
@@ -21,6 +21,9 @@ Version history
 * stream: renamed the `RequestInfo` namespace to `StreamInfo` to better match
   its behaviour within TCP and HTTP implementations.
 * upstream: changed how load calculation for :ref:`priority levels<arch_overview_load_balancing_priority_levels>` and :ref:`panic thresholds<arch_overview_load_balancing_panic_threshold>` interact. As long as normalized total health is 100% panic thresholds are disregarded. 
+* router: added a :ref:`scheme_redirect <envoy_api_field_route.RedirectAction.scheme_redirect>` and
```

Comment:
Last thing: please alphabetize this entry.

Reply:
Done. Thanks.